slash/core/tagging.py

```diff
@@ -46,6 +46,9 @@ def __contains__(self, tag_name):
 
     has_tag = __contains__
 
+    def get_tag(self, tag_name, default_value=None):
+        return self._tags.get(tag_name, default_value)
```

Comment:
default_value should be `NOTHING`, as this is how we denote a missing tag. `None` could be a legitimate tag value